cmd/hope/utils/nodes_test.go

```diff
@@ -83,6 +109,34 @@ func (m *MockHypervisor) UnderlyingNode() (hope.Node, error) {
 	return m.node, nil
 }
 
+func (m *MockHypervisor) CopyImage(a packer.JsonSpec, b hope.VMs, c hope.VMImageSpec) error {
```

Comment:
It's starting to look like the `hypervisor` module should own this, with some configurability via. properties?